frontend/src/lib/gql/gql-client.ts

```diff
@@ -121,10 +122,7 @@ class GqlClient {
 
     if (browser) {
       return derived(resultStore, (result) => {
-        // We can't throw errors in the urql pipeline, because they kill the Svelte application
-        // (Node dies no matter where we throw it, hence the `if (browser)`)
-        setTimeout(() => this.throwAnyUnexpectedErrors(result));
-
+        this.throwAnyUnexpectedErrors(result, true);
```

Comment:
@myieye I refactored this a bit so we're not throwing and catching directly after.